site/content/contribute/expectations/_index.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Contributor expectations"
+heading: "Contributor expectations"
+description: ""
+date: "2022-10-01T12:00:00-00:00"
+weight: 3
+---
+
+## Before contributing
+
+1. If you’re working on code or documentation, you must sign the [Contributor License Agreement](https://mattermost.com/mattermost-contributor-agreement/). This adds you our list of [Mattermost Approved Contributors](https://docs.google.com/spreadsheets/d/1NTCeG-iL_VS9bFqtmHSfwETo5f-8MQ7oMDE5IUYJi_Y/pubhtml?gid=0&single=true). When you provide your mailing address in the signed agreement, you'll receive a [Limited Edition Mattermost Mug](https://forum.mattermost.com/t/limited-edition-mattermost-mugs/143) as a thank you gift after your first pull request is merged.
+2. There are many ways to contribute to Mattermost beyond a core Mattermost repository:
+    a. You can [create an app integration]({{< ref "/integrate/apps/" >}}), which connects to Mattermost’s functionality that's hosted elsewhere.
+    b. Lightweight external applications that don’t require customizations to the Mattermost user experience, can be integrated by creating [incoming]({{< ref "/integrate/webhooks/incoming" >}}) and [outgoing]({{< ref "/integrate/webhooks/outgoing" >}}) webhooks or by using [the Mattermost API](https://api.mattermost.com/).
+    c. You can activate external functionality within Mattermost by creating custom [slash commands]({{< ref "/integrate/slash-commands/" >}}).
+    4. [Plugins]({{< ref "/integrate/plugins/" >}}) enable you to modify the fabric of the Mattermost UI/UX completely but come with the highest level of development overhead and must be written in Go and React.
+    5. If you need to take Mattermost with you into another app, perhaps you’re looking for [the embed guide]({{< ref "/integrate/customization/embedding/" >}}).
+3. Identify which repository you need to work in, then review the README located within the root of the repository to learn more about getting started with your contribution and any processes that may be unique to that repo.
+4. Figure out which repo you need to be working in. With content, you should be able to click the Edit in GitHub button on the published page to find the right repo, but with code it can be somewhat more difficult. You might even have to contribute to multiple repos, likely starting in the server repo. Here are the core repos:
```

Comment:
```suggestion
4. To contribute to documentation, you should be able to get to its repo by clicking the "Edit in Github" button in the top right of its respective published page.
5. Otherwise you can contribute to these Mattermost Core repos:
```

Reply:
Actually thinking we can integrate point 5 into point 2?